test/src/main/java/org/apache/accumulo/test/ComprehensiveTableOperationsIT.java

```diff
@@ -0,0 +1,957 @@
+    // system and user tables
+    Set<String> allTables = ops.list();
+
+    for (String table : allTables) {
+      ops.attachIterator(table, iterSetting, scope);
+      try {
+        assertTrue(ops.listIterators(table).containsKey(iterName));
+        assertNull(ops.getIteratorSetting(table, iterName, IteratorUtil.IteratorScope.scan));
+        assertEquals(iterSetting,
+            ops.getIteratorSetting(table, iterName, IteratorUtil.IteratorScope.majc));
+        assertThrows(AccumuloException.class,
+            () -> ops.checkIteratorConflicts(table, iterSetting, scope));
+        ops.checkIteratorConflicts(table, iterSetting, EnumSet.of(IteratorUtil.IteratorScope.scan));
+      } finally {
+        ops.removeIterator(table, iterName, scope);
+        assertFalse(ops.listIterators(table).containsKey(iterName));
+      }
+    }
+  }
+
+  @Test
+  public void test_addConstraint_listConstraints_removeConstraint() throws Exception {
+    // addConstraint, listConstraints, removeConstraint for user tables is tested in ConstraintIT.
+    // Ensure test exists
+    assertDoesNotThrow(() -> Class.forName(ConstraintIT.class.getName()));
+    // addConstraint, listConstraints, removeConstraint not tested for system tables. Test basic
+    // functionality here
+    for (var sysTable : AccumuloTable.values()) {
+      var numExistingConstraints = ops.listConstraints(sysTable.tableName()).size();
+      String constraint = ComprehensiveBaseIT.TestConstraint.class.getName();
+
+      var constraintNum = ops.addConstraint(sysTable.tableName(), constraint);
+
+      var listConstraints = ops.listConstraints(sysTable.tableName());
+      assertEquals(numExistingConstraints + 1, listConstraints.size());
+      assertEquals(constraintNum, listConstraints.get(constraint));
+
+      ops.removeConstraint(sysTable.tableName(), constraintNum);
+
+      listConstraints = ops.listConstraints(sysTable.tableName());
+      assertEquals(numExistingConstraints, listConstraints.size());
+      assertNull(listConstraints.get(constraint));
+    }
+  }
+
+  @Test
+  public void test_getDiskUsage() throws Exception {
+    // getDiskUsage for user tables is tested in TableOperationsIT. Ensure test exists
+    assertDoesNotThrow(() -> Class.forName(TableOperationsIT.class.getName()));
+    // getDiskUsage not tested for system tables. Test basic functionality here
+    userTable = getUniqueNames(1)[0];
+    ops.create(userTable);
+    createFateTableRow(userTable);
+    createScanRefTableRow();
+    for (var sysTable : AccumuloTable.values()) {
+      ops.flush(sysTable.tableName(), null, null, true);
+      var diskUsageList = ops.getDiskUsage(Set.of(sysTable.tableName()));
+      assertEquals(1, diskUsageList.size());
+      var diskUsage = diskUsageList.get(0);
+      log.info("table : {}, disk usage : {}", sysTable.tableName(), diskUsage.getUsage());
+      if (!sysTable.equals(AccumuloTable.ROOT)) {
+        assertTrue(diskUsage.getUsage() > 0);
+      }
```

Comment:
Skipping ROOT for now until I resolve the bug in TableDiskUsage.getDiskUsage... Will always show a disk usage of 0 for the ROOT table as it is currently scanning the METADATA table for the ROOT table metadata.

Reply:
https://github.com/apache/accumulo/pull/5479 fixed this issue and 4eb99ebd482139c834dbad28948a1842ab88b7d5 removed this skip